andela/ah-shakas-frontend
# Ft see notifications 160577451

## What does this PR do?
* Add notification Functionality
## Description of Task to be completed?
* users should be able to view notifications from the people they follow
## How should this be manually tested?
* clone [this](https://github.com/andela/ah-shakas-frontend.git) repo.
* run command `ft-see-notifications-160577451`.
* run `yarn install` to install requirements
* run `yarn start` to run project
* navigate to [http://localhost:3000/](http://localhost:3000/) locally to access the system locally or [http://10.10.162.103:3000/](http://10.10.162.103:3000/)  to access it online.
## Any background context you want to provide?
* users should be authenticated and also should have people they follow for them to view notifications.

## Packages installed
None

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()

## Todos

## Screenshots (if appropriate)
<img width="518" alt="screenshot 2018-11-30 at 00 57 10" src="https://user-images.githubusercontent.com/23398223/49255171-42c19c80-f43c-11e8-9b67-bc2e67038ae7.png">

## Questions:
<to be deleted>

[sample pr](https://github.com/andela/ah-shakas/pull/39)
